### PR TITLE
add file .editorconfig

### PR DESCRIPTION
**Pull request by icarodgl**
Add .editorconfig for help the patter.

### CODE REVIEW

**Comment by leandromoreira** [comment]
I'm not sure I want to "enforce/suggest" such style... anyway I'll leave to the others.

**Comment by icarodgl** [comment]
change to 4 spaces?

**Comment by mauricioabreu** [comment]
```
indent_style=space
indent_size=4
```

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
```
Comment (mauricioabreu) [inline .editorconfig]: Here you can change to only cover `python` files.

**Pull request merged**